src/Surfnet/ServiceProviderDashboard/Domain/Repository/EntityRepository.php

```diff
@@ -49,4 +49,14 @@ public function findById($id);
      * @return Entity[]
      */
     public function findByServiceId($serviceId);
+
+    /**
+     * @param string $status
+     * @param string $environment
+     * @param int $offset
```

Comment:
Offset and Limit are the other way around. Also, consider removing the offset parameter. It is not used in the application. So why provide the option?